opts.go

```diff
@@ -12,10 +12,14 @@ const (
 	OptLongHelp  = "help"
 	DerefLong    = "dereference-args"
 	StdinName    = "stdin-name"
-	Hidden       = "hidden"
-	HiddenShort  = "H"
-	Ignore       = "ignore"
-	IgnoreRules  = "ignore-rules-path"
+	// FIXME(BLOCKING): Review names. (We may be able to use "output" once
+	//  it's cleaned from subcommands.)
+	OutputFile  = "command-output"
+	ErrorFile   = "command-error"
```

Comment:
Let's keep this simple and use  `--stdout` and `--stderr`  
(self-explanatory + won't clash with `output` or other popular words)

```suggestion
	StdoutFile  = "stdout"
	StderrFile  = "stderr"
```